### PR TITLE
Deployment part is modified

### LINKED ISSUE #306: Improve documentation for how deploys work
**What needs doing?**

Neontribers have requested more detail about how deployments are done.

**Plan**
- [ ] Turn this task into a learning exercise? 
- [ ] Update the acceptance criteria
- [ ] Update the Readme 

**Acceptance Criteria**
- [ ] Neontribers understand how deployments are made

### PR DESCRIPTION
**Pull request by zii-67**
Fixes #306 

## Description

- changed a part of the Readme.md (Deployment)
- I am not really sure what is need to add more, do comment on it so that I can change any.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,10 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
 
 The website is deployed using `now.sh`. The production website reflects the `master` branch. Pull requests which are made against master are considered staging branches.
 
-When contributing to the project branch from master and create a pull request, make sure that changes are approved and that your staging branch works as you expect. **When you merge into master the change will shortly be available on the production site.**
+1. When contributing to the project branch from master and create a pull request, `now.sh` bot will post a status of PR being deployed automatically.
```
Comment (katjam) [inline README.md]: Maybe remove the bit after the comma (`now.sh` bot will post a status of PR being deployed automatically) and include a the status part in  next step?

File: README.md [hunk 2]
```diff
@@ -73,7 +73,10 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
...
+2. `now.sh` will generate a build and deploy in a temporary site which is available to get inside your pull request page.
```
Comment (katjam) [inline README.md]: Maybe something like:

`now.sh` will generate a build and deploy to a temporary url which is available on your pull request page, along with the status of the build

File: README.md [hunk 3]
```diff
@@ -73,7 +73,10 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
...
+3. Make sure that changes are approved and that your staging branch works as you expect.
```
Comment (katjam) [inline README.md]: Could be good to add a 'Before you merge, it is your responsibility to make sure...'

**Comment by zii-67** [comment]
@katjam I have fixed all the changed :smile:

File: README.md [hunk 4]
```diff
@@ -73,7 +73,10 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
 
 The website is deployed using `now.sh`. The production website reflects the `master` branch. Pull requests which are made against master are considered staging branches.
 
-When contributing to the project branch from master and create a pull request, make sure that changes are approved and that your staging branch works as you expect. **When you merge into master the change will shortly be available on the production site.**
+1. Contribute to the project branch from master and create a pull request.
```
Comment (katjam) [inline README.md]: I think it made more sense as before:
`When contributing to the project branch from master and create a pull request.`

File: README.md [hunk 5]
```diff
@@ -73,7 +73,10 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
...
+2. `now.sh` will generate a build and deploy in a temporary url which is available on your pull request page, along with the status of build.
```
Comment (katjam) [inline README.md]:
```suggestion
2. `now.sh` will generate a build and deploy to a temporary url which is available on your pull request page, along with the status of build.
```

Comment (zii-67) [reply README.md]: all done :smiley:

**Comment by RpprRoger** [comment]
Hi! Thanks @zii-67 for taking a look, this is a good improvement. Please add a line item to talk about asking for reviewers or other people to test your code on the staging site created by the pull request.

**Comment by zii-67** [comment]
> Hi! Thanks @zii-67 for taking a look, this is a good improvement. Please add a line item to talk about asking for reviewers or other people to test your code on the staging site created by the pull request.

Hi @RpprRoger Can you please approve it as you've made another PR for this request if I'm correct (and I merged your PR :smile:)

File: README.md [hunk 6]
```diff
@@ -73,7 +73,11 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
 
 The website is deployed using `now.sh`. The production website reflects the `master` branch. Pull requests which are made against master are considered staging branches.
 
-When contributing to the project branch from master and create a pull request, make sure that changes are approved and that your staging branch works as you expect. **When you merge into master the change will shortly be available on the production site.**
+1. When contributing to the project branch from master and create a pull request.
```
Comment (RabidGamer) [inline README.md]: Shouldn't this be "creating a pull request" to match the present participle "contributing"?

File: README.md [hunk 7]
```diff
@@ -73,7 +73,11 @@ Please use `yarn` to manage the dependencies and scripts in this project. You sh
...
+1. `now.sh` will generate a build and deploy to a temporary url which is available on your pull request page, along with the status of build.
```
Comment (RabidGamer) [inline README.md]: "...deploy **it** to a temporary..."

"...status of **the** build..."

**Pull request merged**